_posts/2016-04-08-rules-and-queries.md

```diff
@@ -135,6 +136,9 @@ The backend sends the data on URL you specified in Mergado Developers center in
                 "EAN": "0848719006099",
                 "PRICE": "2525.10",
                 "IMGURL": "https://app.mergado.com/img/890.jpg"
+            },
+            "metadata": {
+                "..."
```

Comment:
Má to nějaké reálné stavy? Nevím jestli je dobré mít tam jako příklad jen 3 tečky.

Reply:
Co si pošlou, to jim vrátíme. Obsah je tedy čistě na nich. Přišlo mi, že by konkrétní případ byl akorát zavádějící.